decred/examples/drain_xpriv.py

```diff
@@ -0,0 +1,255 @@
+#!/usr/bin/env python3
+"""
+Copyright (c) 2020, The Decred developers
+
+This example script will dump all funds located in an xpriv to a specified address.
+"""
+import os
+from urllib.parse import urlunsplit
+
+from base58 import b58decode, b58encode
+from decred import DecredError
+from decred.crypto import crypto
+from decred.crypto.secp256k1.curve import curve as Curve
+from decred.dcr import account, addrlib, nets, rpc, txscript
+from decred.dcr.wire import msgtx, wire
+from decred.util import helpers
+from decred.util.encode import ByteArray
+
+
+SERIALIZED_KEY_LENGTH = 4 + 1 + 4 + 4 + 32 + 33  # 78 bytes
+INTERNAL = 0
+EXTERNAL = 1
+FEE_RATE = 10000
+GAP_LIMIT = 20
+
+
+def cfg(isTestnet):
```

Comment:
This function appears identical to [this one](https://github.com/decred/tinydecred/blob/280dc57df991d0b2125f4161011afd568cb40f0a/decred/tests/conftest.py#L53), and about the same as [this one](https://github.com/decred/tinydecred/blob/280dc57df991d0b2125f4161011afd568cb40f0a/tinywallet/tinywallet/config.py#L113). Maybe this function deserves a promotion to a common module like `helpers` or something. Not a perfect fit, but better than triplication.

Reply:
Ok to fix in a future pr? I also need a check for txindex being enables for this...

Reply:
maybe just addrindex? not really sure.

Reply:
checking both, I'm sure there's a way to write it so that we can do everything with one method though.